speaker/main.go

```diff
@@ -343,6 +345,19 @@ func poolFor(pools map[string]*config.Pool, ip net.IP) string {
 	return ""
 }
 
+func (c *controller) bgpImplementationSupportAggrLenV6Cfg(pools map[string]*config.Pool) bool {
+	if c.bgpType == bgpNative {
+		for _, p := range pools {
+			for _, adCfg := range p.BGPAdvertisements {
+				if adCfg.AggregationLengthV6 != 0 && adCfg.AggregationLengthV6 != 128 {
```

Comment:
This would make 128 a valid value (even for the native implementation), which is not desirable. 
I'd rather apply the default in `SetConfig` and leave it empty when we read the config. 

Something like a "SetFRRDefaults" that we call over the configuration object if c.bgpType == bgpNative.

It's not symmetric with what we do with AggregationLength, but I think it's cleaner and we'll need it for other frr-only values.

Reply:
this won't work becasue there is a little check in the config to make sure sure this length isn't less than cidr mask len, I will think about away out

Reply:
This is as ugly. 
Let's not fail if the user sets a v6 aggregation lenght, let's fail if they set a v6 cidr in native mode.

Reply:
there were many CI test cases when I just check for V6 I will defer this check for future PR instead of causing functional breakage or CI issues